app/app.css

```diff
@@ -55,7 +101,7 @@ h1 {
 }
 
 .connection {
-    height: 400px;
+    height: 475px;
 }
```

Comment:
Just drop this hard-coded value, the form container have a different height with the "WebComponents not supported" message ; the buttons overflow in this case.